CHANGELOG

```diff
@@ -1,3 +1,6 @@
+* Tue Sep 11 2018 Steven Pritchard <steven.pritchard@onyxpoint.com> - 6.3.1-0
```

Comment:
6.3.0 has not released, so this version bump is not appropriate.  Please set the version to 6.3.0

Reply:
Done.